lua/package-info/utils/job.lua

```diff
@@ -23,7 +23,15 @@ return function(props)
         end
     end
 
+    -- If a package.json file is open, use the directory of the file
+    -- as working directory for the job
+    local cwd = vim.fn.getcwd()
+    if string.sub(vim.fn.expand("%:p"), -12) == "package.json" then
```

Comment:
Why the `-12` ? Can you elaborate on that? Same for `-13`.

Reply:
`vim.fn.expand("%:p")` returns a string containing the full path to the opened file including the file name. `string.sub()` uses indexes to define where to start and where to end extracting sub strings. Those indexes can be negative which means that the function will start counting from the back. The first call extracts everything from position `-12` to the end of the string. The string `package.json` is 12 characters long hence the `-12`. The second call extracts everything from the start to position `-13` which is the path to the file or the directory the file is in.